source/guides/clear/swupd-3rd-party.rst

```diff
@@ -0,0 +1,284 @@
+.. _swupd-3rd-party:
+
+Create, Install, and Manage 3rd-party Bundles
```

Comment:
Suggest simplifying the title to `swupd 3rd-party bundles` to be inline with the title naming of the rest of the guides. The verbs can be in the body.

Reply:
Is your suggestion only applicable to "Guides" section?  We have other docs. that doesn't follow.

Reply:
I think so., especially under the Clear Linux section. I'm simply looking at the navigation: https://docs.01.org/clearlinux/latest/guides/index.html#clear-linux

I took this approach with the tutorial section and it looks significantly more neat.

Reply:
done.